addons/dragon/CfgVehicles.hpp

```diff
@@ -0,0 +1,207 @@
+class MainTurret;
+class CfgVehicles {
+    class LandVehicle;
+    
+    class StaticWeapon : LandVehicle {
+        class Turrets {
+            class MainTurret;
+        };
+    };
+    
+    class StaticMGWeapon: StaticWeapon{
+        class Turrets : Turrets  {
+            class MainTurret : MainTurret {
+                class ViewOptics;
+            };
+        };
+    };
+    
+    class AT_01_base_F: StaticMGWeapon {
+        class Turrets : Turrets  {
+            class MainTurret : MainTurret {
+                class ViewOptics: ViewOptics {
+                };
+            };
+        };
+    };
+    
+    class GVAR(dragonStaticBase) : AT_01_base_F {
```

Comment:
inheriting from existing things can cause problems down the line if another mod changes them
inheriting from `class StaticATWeapon: StaticWeapon {` would probably be better, but I'm not sure what that would require